src/styles/components/_text-button.scss

```diff
@@ -0,0 +1,30 @@
+@use '../abstracts/variables' as variables;
+@use '../abstracts/colors' as colors;
+@use '../abstracts/mixins' as mixins;
+
+.text-button {
```

Comment:
we can use BEM naming here:

.button // contains base style
  .button__text // contains more specific styles
  .button__fill


usage:
<button class="button button__fill"